homeassistant/components/binary_sensor/netatmo.py

```diff
@@ -49,48 +62,68 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     netatmo = get_component('netatmo')
     home = config.get(CONF_HOME, None)
     timeout = config.get(CONF_TIMEOUT, 15)
+    offset = config.get(CONF_OFFSET, 90)
```

Comment:
- **timeout** (*Optional*): The Welcome binary sensors will reflect the events from the last X minutes. (default: 15)
- **offset** (*Optional*): The Presence binary sensors will stay on for X seconds after detection. (default: 90)